app/Character/Factory/CharacterFactory.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Noblesse\Character;
+
+require_once $_SERVER['DOCUMENT_ROOT'] .'vendor/autoload.php';
+
+use Noblesse\Character\Character;
+
+$char = new Character('asd','ad','asd');
+
+$char->damage = [10, 30];
+$stack = $char->damage;
+array_push($stack, 20);
+$char->damage = $stack;
+
+var_dump($char->damage);
```

Comment:
is this necessary to have var_dump? or you can use echo or print to display certain attributes of the object?